ethers-ext/example/transactions/Basic_20_TxTypeAccountUpdate.js

```diff
@@ -27,7 +26,7 @@ async function main() {
         from: senderAddr,
         key: {
             type: 0x02, 
-            key: crypto.computePublicKey(senderNewPriv, true),
+            key: wallet.computePublicKey(senderNewPriv, true),
```

Comment:
There's `ethers.utils.computePublicKey`. https://docs.ethers.org/v5/api/utils/signing-key/
It's awkward for a Wallet to have this method.

Reply:
Good comment. 
I will use original function ethers.utils.computePublicKey.